src/AutomatedCar/SystemComponents/Packets/HMIPacket.cs

```diff
@@ -150,5 +151,45 @@ public void AccSet(bool on)
                 this.acc = !this.acc;
             }
         }
+
+        public void HandleGasPedal(double tick, bool isGasPedalDown)
+        {
+            if (isGasPedalDown)
+            {
+                Increase(tick, ref this.gaspedal);
+            }
+            else
+            {
+                Decrease(tick, ref this.gaspedal);
+            }
+        }
+
+        public void HandleBrakePedal(double tick, bool isBrakePedalDown)
+        {
+            if (isBrakePedalDown)
+            {
+                Increase(tick, ref this.gaspedal);
+            }
+            else
+            {
+                Decrease(tick, ref this.gaspedal);
+            }
+        }
+
+        private void Increase(double tick, ref double pedal)
+        {
+            while (pedal != 100)
+            {
+                pedal += 1 / 60;
+            }
```

Comment:
Megint itt ez egy ciklus és addig megy ameddig nem lesz a pedal 100
És én is rosszul írtam az előbb 10/6 -al kell növelni, hogy 60 tick alatt elérje a 100 as értéket.
Szerintem elég lenne egy 
```
if (pedal < 100)
{
    pedal += 10/6;
}
```
És nem tudom hogy futás közben mi mehet félre de lehet hogy vizsgálni kéne hogy az érték ne menjen 100 felé és 0 alá

Reply:
aah ertem, okes, felreneztem akkor en is, javitom ezt maris (valamiert eddig azt hittem nem minden ticknel fut le, amiatt volt igy megvalositva)